### PR TITLE
window size bug fix and supporting appium lib from 11.2

### PR DESCRIPTION
**Pull request by prklm10**
fixing no method error get_window_size and adding compatibility from Appium lib version 11.2

### CODE REVIEW

File: specs/ios_metadata.rb [hunk 1]
```diff
@@ -37,7 +37,7 @@ def test_get_window_size
     height = 100
     width = 100
     window_size = { 'height' => height, 'width' => width }
-    @mock_webdriver.expect(:get_window_size, window_size)
```
Comment (ninadbstack) [inline specs/ios_metadata.rb:40]: Shouldnt we ideally support both ?

Comment (prklm10) [reply specs/ios_metadata.rb:40]: there is no function in ruby called `get_window_size` docs:- https://appium.readthedocs.io/en/stable/en/commands/web/window/get-window-size/.
it is a function in python. Maybe when we converted it form python to ruby we didn't test this.
Even our example project has device that is present in device.json file.

Comment (prklm10) [reply specs/ios_metadata.rb:40]: yes.

Comment (ninadbstack) [reply specs/ios_metadata.rb:40]: Can you revert that then ?

Comment (prklm10) [reply specs/ios_metadata.rb:40]: appium lib > 11.2 is a fix for another customer issue

**Review by ninadbstack — APPROVED** [review]
LGTM!

**Pull request merged**